config/templates/wp-config.php.erb

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * The base configuration for WordPress
+ *
+ * The wp-config.php creation script uses this file during the
+ * installation. You don't have to use the web site, you can
+ * copy this file to "wp-config.php" and fill in the values.
+ *
+ * This file contains the following configurations:
+ *
+ * * MySQL settings
+ * * Secret keys
+ * * Database table prefix
+ * * ABSPATH
+ *
+ * @link https://codex.wordpress.org/Editing_wp-config.php
+ *
+ * @package WordPress
+ */
+
+
+/******************** ChapPress - MySQL settings **************************/
+
+/** These database values are only to be used for local environment 
+ *  development for the ChapPress project.
+ */
+
+/** The name of the database for WordPress */
+define('DB_NAME',     '<%= database['database'] %>');
```

Comment:
I don't think comments are needed for these settings. Var names are clear enough. You can just list them:

```
define('DB_NAME',     '<%= database['database'] %>');
define('DB_USER',     '<%= database['username'] %>');
...
```